benchmarks/GraphBLAS-sharp.Benchmarks/Helpers.fs

```diff
@@ -267,3 +267,61 @@ module Utils =
             rowPointers.[i] <- rowPointers.[i - 1] + nnzPerRow.[i - 1]
 
         rowPointers
+
+module StandardOperations =
+    let boolSum =
+        <@ fun (_: bool option) (_: bool option) -> Some true @>
+
+    let intSum =
+        <@ fun (x: int option) (y: int option) ->
+            let mutable res = 0
+
+            match x, y with
+            | Some f, Some s -> res <- f + s
+            | Some f, None -> res <- f
+            | None, Some s -> res <- s
+            | None, None -> ()
+
+            if res = 0 then None else (Some res) @>
+
+    let byteSum =
+        <@ fun (x: byte option) (y: byte option) ->
+            let mutable res = 0uy
+
+            match x, y with
+            | Some f, Some s -> res <- f + s
+            | Some f, None -> res <- f
+            | None, Some s -> res <- s
+            | None, None -> ()
+
+            if res = 0uy then None else (Some res) @>
+
+    let floatSum =
+        <@ fun (x: float option) (y: float option) ->
+            let mutable res = 0.0
+
+            match x, y with
+            | Some f, Some s -> res <- f + s
+            | Some f, None -> res <- f
+            | None, Some s -> res <- s
+            | None, None -> ()
+
+            if abs (res) < 1E-08 then
```

Comment:
But why. I think, if one use floats, he knows about precision. I guess, we should use comparison with zero.